Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workbench/PublicPage.java

```diff
@@ -0,0 +1,121 @@
+package edu.asu.spring.quadriga.domain.impl.workbench;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @description : Project class describing the properties of a Public Page object
+ * 
+ * @author : Prasanth Priya Nesan
+ * 
+ */
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.domain.workbench.IPublicPage;
+
+public class PublicPage implements IPublicPage {

```

Comment:
yes we will create separate class for the title/description pairs and give a pull request